compiler/rustc_borrowck/src/diagnostics/bound_region_errors.rs

```diff
@@ -360,6 +362,7 @@ fn try_extract_error_from_fulfill_cx<'tcx>(
     ocx: &ObligationCtxt<'_, 'tcx>,
     placeholder_region: ty::Region<'tcx>,
     error_region: Option<ty::Region<'tcx>>,
+    generic_param_scope: LocalDefId,
```

Comment:
Regarding this and many other callsites, I slightly have a nit to prefer putting `generic_param_scope` closer to the front 🤔